azimuth/routers/v1/utterances.py

```diff
@@ -264,6 +265,47 @@ def get_utterances(
     )
 
 
+@router.post(
+    "",
+    summary="Patch utterances",
+    description="Patch utterances, such as updating proposed actions.",
+    tags=TAGS,
+    response_model=List[UtterancePatch],
+)
+def patch_utterances(
+    request_data: List[UtterancePatch] = Body(...),
+    dataset_split_manager: DatasetSplitManager = Depends(get_dataset_split_manager),
+    task_manager: TaskManager = Depends(get_task_manager),
+) -> List[UtterancePatch]:
+    persistent_ids = [utt.persistent_id for utt in request_data]
```

Comment:
Like, how beautiful would that be? 😍 
```suggestion
    persistent_ids = [utterance.persistent_id for utterance in utterances]
```

Reply:
Haha sold. I did think `request_data` was maybe standard for a body. Thank you!!